lib/foreman_tasks/tasks/export_tasks.rake

```diff
@@ -221,6 +222,15 @@ namespace :foreman_tasks do
         tasks.order('started_at desc').all.each do |task|
           html << "<tr><td><a href=\"#{task.id}.html\">#{task.label}</a></td><td>#{task.started_at}</td>\
                    <td>#{task.state}</td><td>#{task.result}</td></tr>"
+        rescue StandardError => e
+          if skip_errors
```

Comment:
This runs in a different context than where `skip_errors` is defined

```
NameError: undefined local variable or method `skip_errors' for PageHelper:Class

/home/aruzicka/projects/foreman/foreman-tasks/lib/foreman_tasks/tasks/export_tasks.rake:227:in `rescue in block in generate_index'
/home/aruzicka/projects/foreman/foreman-tasks/lib/foreman_tasks/tasks/export_tasks.rake:223:in `block in generate_index'
/home/aruzicka/projects/foreman/foreman-tasks/lib/foreman_tasks/tasks/export_tasks.rake:222:in `generate_index'
/home/aruzicka/projects/foreman/foreman-tasks/lib/foreman_tasks/tasks/export_tasks.rake:282:in `block (4 levels) in <main>'
/home/aruzicka/projects/foreman/foreman-tasks/lib/foreman_tasks/tasks/export_tasks.rake:282:in `open'
/home/aruzicka/projects/foreman/foreman-tasks/lib/foreman_tasks/tasks/export_tasks.rake:282:in `block (3 levels) in <main>'
/home/aruzicka/projects/foreman/foreman-tasks/lib/foreman_tasks/tasks/export_tasks.rake:261:in `block (2 levels) in <main>'
```

Reply:
Hopefully fixed this with the latest commit.